### ISSUE TITLE
memory error

### ISSUE THREAD

**Issue by 94zsh** [Zero-to-Heroes/firestone#334]
<img width="876" alt="斗鱼直播截图_20240128224554" src="https://github.com/Zero-to-Heroes/firestone/assets/20756780/ba9cc533-5e67-439d-a7d7-4a55525919ba">
![image](https://github.com/Zero-to-Heroes/firestone/assets/20756780/3e19f004-29f0-4269-9985-fec2653ea3a0)
memory error , alread admin

**Comment by sebastientromp** [comment]
Is hearthstone running as admin as well? If so, could you try to remove the admin privileges from the game?
Also please note that you might have to restart Windows after making the changes

**Comment by 94zsh** [comment]
not work，Please refer to the screenshot，game is remove the admin privileges，fireston 、overwolf is running as addmin，Restarting the computer is useless
<img width="366" alt="斗鱼直播截图_20240129193734" src="https://github.com/Zero-to-Heroes/firestone/assets/20756780/4d6d233b-7fca-444c-826d-aee06e0779a6">
<img width="326" alt="斗鱼直播截图_20240129193806" src="https://github.com/Zero-to-Heroes/firestone/assets/20756780/2481dfa3-a741-4103-989d-78d780c497ff">
<img width="325" alt="斗鱼直播截图_20240129193817" src="https://github.com/Zero-to-Heroes/firestone/assets/20756780/a08a24ba-596f-454d-80f3-ca12d0a10a9a">

**Comment by 94zsh** [comment]
I used it for a long time without any problems, but since the last update of Hearthstone, nothing has been done and it has been problematic

**Comment by 94zsh** [comment]
<img width="918" alt="斗鱼直播截图_20240129194941" src="https://github.com/Zero-to-Heroes/firestone/assets/20756780/0467d7f4-7fa5-45f8-9ecd-24b4d4836582">

**Comment by sebastientromp** [comment]
Thanks. This error occurs when the app doesn't have the rights to read the game client's memory, so I suspect something must have changed somewhere.
Could you maybe have an antivirus or something that could restrict the app's access?

**Comment by 94zsh** [comment]
My computer is Win11, Lenovo, and there are no abnormalities in the antivirus software. I have also tried to turn off the protection software, but still have an error reading memory. At the same time, most of the time, it is a memory reading error, and in rare cases, memory can be read normally. In this case, restarting the game will cause another memory error

**Comment by 94zsh** [comment]
Thank you. After the latest game update, the plugin usage has returned to normal. This may be due to the game fixing this issue

**Comment by wangzhengyr** [comment]
I had the same problem
![image](https://github.com/user-attachments/assets/c9f2a0eb-5e35-4030-b798-28fbec84bc08)

**Comment by wangzhengyr** [comment]
> 感谢您的报告！您是否也尝试过以管理员身份运行 Overwolf（不仅仅是 FIrestone）并在那之后重新启动您的 PC？

![image](https://github.com/user-attachments/assets/82a66198-fbc8-4606-ba15-48eb3ff77ce5)
![image](https://github.com/user-attachments/assets/ac48c16f-8604-4b38-be08-5b79467108aa)

yea but still an error

**Comment by sebastientromp** [comment]
Do you maybe have an antivirus / firewall that could prevent Firestone from reading the game client's memory?

**Comment by wangzhengyr** [comment]
> Do you maybe have an antivirus / firewall that could prevent Firestone from reading the game client's memory?

No, I don't have any antivirus or firewall that could prevent Firestone from reading the game client's memory.

**Comment by sebastientromp** [comment]
Sorry for the lack of updates, I have not forgotten about you. Trying to think about what could be causing the issue.  
Have you checked all the steps from https://github.com/Zero-to-Heroes/firestone/wiki/FAQ---Troubleshooting#memory-reading-error--run-as-admin ?

**Comment by wangzhengyr** [comment]
> Sorry for the lack of updates, I have not forgotten about you. Trying to think about what could be causing the issue. Have you checked all the steps from https://github.com/Zero-to-Heroes/firestone/wiki/FAQ---Troubleshooting#memory-reading-error--run-as-admin ?

I did it. Thank you very much. The unicode of my previous system was English, but now it can be changed to Chinese which is consistent with the system
